setup.py

```diff
@@ -17,55 +16,7 @@ def get_version():
 }
 extras["all"] = sum(extras.values(), [])
 
-install_requires = open("requirements.txt").read().strip().split("\n")
```

Comment:
I really hate one-line open's, open should always be with `with` - and CPython will spit warnings about this if you enable all warnings. Nice to get rid of this one - but there's still one above in get_version.

Reply:
I've gone ahead and removed it too.

Reply:
Thanks.